app/local.properties

```diff
@@ -1,8 +0,0 @@
-## This file must *NOT* be checked into Version Control Systems,
```

Comment:
I deleted this file since this should not be at all in the VCS, since it contains local machine settings